kubectl_plugins/bind-service/bind-service.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/kubernetes-incubator/service-catalog/kubectl_plugins/utils"
+	v1alpha1 "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1alpha1"
+	clientset "github.com/kubernetes-incubator/service-catalog/pkg/client/clientset_generated/clientset"
+
+	"k8s.io/client-go/pkg/api/v1"
+	"k8s.io/client-go/rest"
+)
+
+const USAGE = `
+
+Usage:
+  kubectl plugin bind-service SERVICE_CLASS_NAME SECRET_NAME NAMESPACE/INSTANCE_NAME
```

Comment:
As of right now, we are not able to leverage the global option flag `--namespace`. While kubectl will accept the flag, it is not being passed on the plugin. I have made "namespace" into an argument, with the intention to eventually make it into an option flag, I believe kubectl will soon expose this feature (work is being done).

Regarding your comment of binding service not needing a service class, I wasn't able to get it to work without provide the service class name. This is the error I get without one:
```
Error binding service instance (Binding.servicecatalog.k8s.io "ups-binding" is invalid: Spec.instanceRef.name: Invalid value: "": a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*'))
```